src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -115,6 +118,7 @@ export class BaselinesTable extends Component {
                     tableId={ tableId }
                     baselineRowData={ baseline }
```

Comment:
You can remove this parameter now that it's not being used by BaselineTableKebab.